.idea/modules.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Orthanc-TMI.iml" filepath="$PROJECT_DIR$/.idea/Orthanc-TMI.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/Orthanc_TMI.iml" filepath="$PROJECT_DIR$/.idea/Orthanc_TMI.iml" />
     </modules>
   </component>
```

Comment:
Not sure how any `.idea/` files got in. I thought the directory was ignored in .gitignore.

We should delete this file and any others in there.